go-controller/pkg/ovn/egressip.go

```diff
@@ -363,7 +363,6 @@ type egressIPMode struct {
 
```

Comment:
The commit message isn't really accurate:
"With the changes to "new local gateway" mode (which removed gateway routers)
we need to retrieve the gateway router IP using the logical router port instead."

It doesn't remove GRs, it just doesnt force snat on them anymore.

Reply:
Yes, I missed that. I will re-write the commit